### PR TITLE
docs: Session Recording Requirements for Workers

### PR DESCRIPTION
**Pull request by elimt**
Clarify the requirements for Boundary workers used for session recording

### CODE REVIEW

File: website/content/docs/configuration/session-recording/create-storage-bucket.mdx [hunk 1]
```diff
@@ -19,14 +19,14 @@ Any storage buckets associated with an Org scope are deleted when the Org itself
 
 For more information about using session recording to audit user sessions, refer to [Auditing](/boundary/docs/concepts/auditing).
 
-**Requirements**:
+## Requirements
+
+### AWS Requirements
+
```
Comment (Dan-Heath) [inline website/content/docs/configuration/session-recording/create-storage-bucket.mdx:25]:
```suggestion
## Requirements

Before you can create a storage bucket, you must ensure that your environment meets certain requirements.
Session recording requires specific configuration for both the AWS S3 storage bucket and the Boundary worker.

### AWS requirements

```

File: website/content/docs/configuration/session-recording/create-storage-bucket.mdx [hunk 2]
```diff
@@ -54,6 +54,24 @@ For more information about using session recording to audit user sessions, refer
    }
    ```
 
+### Boundary Workers Requirements
```
Comment (Dan-Heath) [inline website/content/docs/configuration/session-recording/create-storage-bucket.mdx:57]:
```suggestion
### Boundary workers requirements
```

File: website/content/docs/configuration/session-recording/create-storage-bucket.mdx [hunk 3]
```diff
@@ -54,6 +54,24 @@ For more information about using session recording to audit user sessions, refer
...
+
+[Session recording](/boundary/docs/configuration/session-recording) requires at least one PKI worker with access to local and remote storage.
+- A Boundary PKI worker with access to the AWS S3 storage bucket
+- A Boundary PKI worker with an accessible directory defined by `recording_storage_path` for storing in-progress session recordings. On session closure, a local session recording is moved to remote storage and deleted locally. For more details, refer to [PKI workers](/boundary/docs/configuration/worker/pki-worker#session-recording-hcp-ent).
+- A Boundary PKI worker with a minimum of 1 MB available disk space.
+- A Boundary PKI worker running on darwin, windows or linux. The following binaries are not supported for session recording: nestbsd, openbsd, solaris
```
Comment (Dan-Heath) [inline website/content/docs/configuration/session-recording/create-storage-bucket.mdx:63]:
```suggestion
[Session recording](/boundary/docs/configuration/session-recording) requires at least one PKI worker that:

- Has access to the AWS S3 storage bucket
- Has an accessible directory defined by `recording_storage_path` for storing session recordings while they are in progress. On session closure, Boundary moves the local session recording to remote storage and deletes the local copy. For more details, refer to [PKI workers](/boundary/docs/configuration/worker/pki-worker#session-recording-hcp-ent).
- Has at least 1 MB of available disk space.
- Runs Darwin, Windows, or Linux. The following binaries are not supported for session recording: NetBSD, OpenBSD, Solaris.
```

Comment (Dan-Heath) [reply website/content/docs/configuration/session-recording/create-storage-bucket.mdx:63]: I tried to consolidate this bulleted list so we don't have to repeat "A Boundary PKI worker" at the beginning of each line. But please make sure I didn't change the meaning.

**Review by Dan-Heath — CHANGES_REQUESTED** [review]
This is a great addition to this topic, thank you @elimt ! I like breaking these up into AWS and Boundary worker requirements. I suggested some minor copy edits. Please let me know if you have any questions, or if my edits don't make sense.

**Comment by AdamBouhmad** [comment]
> This is a great addition to this topic, thank you @elimt ! I like breaking these up into AWS and Boundary worker requirements. I suggested some minor copy edits. Please let me know if you have any questions, or if my edits don't make sense.

To further separate these, do we think it makes sense to split these out into: 

Overview 
Worker Configuration 
Create a storage bucket 
Enable session recording

**Comment by Dan-Heath** [comment]
> To further separate these, do we think it makes sense to split these out into:
> 
> Overview
> Worker Configuration
> Create a storage bucket
> Enable session recording

Our template for how-to guide procedures like this recommends that we keep pre-requisites in a single location to help readers prepare their environments to perform the tasks described in the topic. I'd be afraid readers might miss those pre-reqs, if we moved them to a new topic.

**Review by Dan-Heath — APPROVED** [review]
This looks great to me, thank you Elim!

**Pull request merged**